### ISSUE TITLE
undefined model for decorators

### ISSUE THREAD

**Issue by aksonov** [mobxjs/serializr#2]
```
class File {
  @serializable(primitive())
  id: string;
  @serializable(primitive())
  source: string;

  constructor(id: string, source:string){
    this.id = id;
    this.source = source;
  }
}
console.log(File.serializeInfo);
```

return undefined, strange
but when i put log inside serializableDecorator, it returns correct object

```
File {
  serializeInfo: { factory: [Function], props: { id: [Object] } } }
File {
  serializeInfo: 
   { factory: [Function],
     props: { id: [Object], source: [Object] } } }
undefined
```

i’ve added log within decorator:

```
             info.props[propName] = propSchema
            console.log(target);
```
any idea what it could be? as the result i can’t define complex structures...

**Comment by Bnaya** [comment]
I have similar issue,
It's looks like it sets the serializeInfo on the prototype of the constructor and not the constructor itself

**Comment by mweststrate** [comment]
Released as 0.0.2. @aksonov does this solve the issue for you as well?
